examples/coprocessor_java/robot/build.gradle

```diff
@@ -16,9 +16,20 @@ deploy {
     }
 }
 
+// Note - the robot code doesn't need these repositories.
+// However, the 'common' project does, and for some reason it won't pull
+// its dependencies unless the repository is listed here.  Keep that in mind.
+repositories {
+  mavenCentral() // Used for the GSON library (Google's JSON converter)
```

Comment:
Please move the mavenCentral() declaration from all projects (robot, common, coprocessor) to the root build.gradle in the subprojects {} block, since it's being applied to everything

Reply:
Done.